common/src/main/java/com/microsoft/identity/common/internal/cache/CacheKeyValueDelegate.java

```diff
@@ -175,6 +177,14 @@ public String generateCacheKey(Credential credential) {
                 cacheKey = cacheKey.replace(AUTH_SCHEME, sanitizeNull(accessToken.getAccessTokenType()));
             }
 
+            if(!StringExtensions.isNullOrBlank(accessToken.getRequestedClaims()))
```

Comment:
Minor formatting nit: can we move the curly-brace to previous line?

Reply:
Fixed in latest commit